app/src/main/java/org/mozilla/focus/fragment/DownloadsFragment.java

```diff
@@ -53,7 +52,9 @@ public void onReceive(Context context, Intent intent) {
                         public void onQueryComplete(List downloadInfoList) {
                             for (int i=0;i<downloadInfoList.size();i++){
                                 DownloadInfo downloadInfo = (DownloadInfo) downloadInfoList.get(i);
-                                mDownloadListAdapter.updateItem(downloadInfo);
+                                if (mDownloadListAdapter != null){
```

Comment:
The BroadcastReceiver is created after creation of DownloadListAdapter, how come the adapter becomes null?